kubeedge-web-demo/README.md

```diff
@@ -0,0 +1,107 @@
+# KubeEdge Web Demo
+
+Firstly the users input `kubeedge` to get the web app link,
```

Comment:
where do they need to provide this input ? in a browser, app ?

Reply:
Updated. Thanks @rohitsardesai83